src/callstack.cr

```diff
@@ -130,16 +130,17 @@ struct CallStack
     frame = decode_frame(repeated_frame.ip)
     if frame
       offset, sname = frame
+      sname = String.new sname
       if repeated_frame.count == 0
-        LibC.printf "[%ld] %s +%ld\n", repeated_frame.ip, sname, offset
+        STDERR.printf "[%d] %s +%d\n", repeated_frame.ip.address, sname, offset
```

Comment:
@RX14 thanks for the tip, I've updated the PR accordingly. I was thinking of introducing an `enum (STDIN, STDOUT, STDERR)` for the `fd` argument to improve readability, WDYT?